binding/nodejs/test/test_utils.ts

```diff
@@ -1,6 +1,9 @@
 import * as path from "path";
 import {getPlatform} from "../src/platforms";
 
+const ROOT_DIR = path.join(__dirname, "../../..");
```

Comment:
for each `get*` method in test utils you can remove the `__dirname` and `relative` param and use this. The tests all just pass in `../../../` for the `relative` arg